zebra-chain/src/parameters/network.rs

```diff
@@ -274,6 +276,54 @@ impl Network {
             .activation_height(self)
             .expect("Sapling activation height needs to be set")
     }
+
+    /// Returns the expected total value of the sum of all NU6.1 one-time lockbox disbursement output values for this network at
+    /// the provided height.
+    pub fn lockbox_disbursement_total_amount(&self, height: Height) -> Amount<NonNegative> {
+        if Some(height) != NetworkUpgrade::Nu6_1.activation_height(self) {
+            return Amount::zero();
+        };
+
+        match self {
+            Self::Mainnet => subsidy::EXPECTED_NU6_1_LOCKBOX_DISBURSEMENT_TOTAL_MAINNET,
+            Self::Testnet(params) if params.is_default_testnet() => {
+                subsidy::EXPECTED_NU6_1_LOCKBOX_DISBURSEMENT_TOTAL_TESTNET
+            }
```

Comment:
The relationship between `EXPECTED_NU6_1_LOCKBOX_DISBURSEMENT_TOTAL_MAINNET` and the sum of `NU6_1_LOCKBOX_DISBURSEMENT_MAINNET` should be enforced by a test at very least; however, it might be better to just compute the total value so that there's not a possibility of error.